website/manual.md

```diff
@@ -1155,7 +1155,7 @@ We are very concerned about making mistakes when adding new APIs. When adding an
 Op to Deno, the counterpart interfaces on other platforms should be researched.
 Please list how this functionality is done in Go, Node, Rust, and Python.
 
-As an example, see how `Deno.rename()` was proposed and added in
+As an example, see how `deno.rename()` was proposed and added in
```

Comment:
No, it is `Deno.rename()`, not `deno.rename()`.

Reply:
Your right sorry, I just looked at the name in the original issue header should have doubled checked